### ISSUE TITLE
sklearn quantreg is bad on categorical data ( after one hot) i wrote them but thye are not intersted ....

### ISSUE THREAD

**Issue by Sandy4321** [avidale/anekbot-example#1]
sklearn quantreg is bad on categorical data ( after one hot) i wrote them but thye are not intersted ....

avidale/quantreg_gradient.ipynb

Here is a naive implementation of gradient descent for Quantile regression problem (scikit-learn/scikit-learn#9978)

**Comment by avidale** [comment]
Sorry, this repo is 100% irrelevant for your comment.

**Comment by Sandy4321** [comment]
do you use
Quantile regression?

**Comment by avidale** [comment]
No, I don't use it. 
I once implemented it for scikit-learn, but I am not actively supporting it.

**Comment by Sandy4321** [comment]
I once implemented it for scikit-learn
where is code?

**Comment by avidale** [comment]
In scikit-learn, where else can it be?
https://scikit-learn.org/stable/auto_examples/linear_model/plot_quantile_regression.html

**Comment by Sandy4321** [comment]
it is what I mentioned above 
Here is a naive implementation of gradient descent for Quantile regression problem (https://github.com/scikit-learn/scikit-learn/pull/9978)
also i reported to them bad performance for one hot data - they do not care

**Comment by avidale** [comment]
I don’t care either, sorry.
